src/components/RadioButton/RadioButton.driver.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { UniDriver } from 'wix-ui-test-utils/unidriver';
+import { RADIOBUTTON_DATA_KEYS } from './datahooks';
+
+export interface RadioButtonDriver extends BaseUniDriver {
+  hasChecked(): Promise<boolean>;
+  hasDisabled(): Promise<boolean>;
```

Comment:
Lets change these to 
```suggestion
  isChecked(): Promise<boolean>;
  isDisabled(): Promise<boolean>;
```